qemu/tests/uefi_check_resolution.py

```diff
@@ -0,0 +1,82 @@
+import re
+import random
+
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify UEFI config setting in the GUI screen:
+    1) Boot up a guest.
+    2) Check default GUI setting
+    3) Change 'Change Prefered' to $re1 in list
+    4) Save it by hitting 'F10'+'Y'
+    5) Restart vm
+    6) Check if 'Change Prefered' had been change to $re1
+    7) Change 'Change Prefered' to $re2 in list
+    8) Save it by hitting "Commit Changes and Exit"
+    9) Restart vm
+    10) Check if 'Change Prefered' had been change to $re2
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    ChangePrefered = ["640 x 480", "800 x 480", "800 x 600", "832 x 624",
+                      "960 x 640", "1024 x 600", "1024 x 768", "1152 x 864",
+                      "1152 x 870", "1280 x 720", "1280 x 760", "1280 x 768",
+                      "1280 x 800", "1280 x 960", "1280 x 1024", "1360 x 768",
+                      "1366 x 768", "1400 x 1050", "1440 x 900", "1600 x 900",
+                      "1600 x 1200", "1680 x 1050", "1920 x 1080",
+                      "1920 x 1200", "1920 x 1440", "2000 x 2000",
+                      "2048 x 1536", "2048 x 2048", "2560 x 1440",
+                      "2560 x 1600"]
+
+    def execute_key(keys):
+        """
+        send key
+        """
+        for i in keys:
+            vm.send_key(i)
+
+    def boot_check(info):
+        """
+        boot info check
+        """
+        logs = vm.logsessions['seabios'].get_output()
+        result = re.search(info, logs, re.S)
+        return result
+
+    def set_change_step(change_config_key):
+        """
+        set key to change gui config
+        """
+        n = random.randint(1, 10)
+        change_config_key = change_config_key[0:1] + ["down"] * n + ["kp_enter"]
+        resolution = ChangePrefered[n]
+        check_info = "GraphicsConsole video resolution " + resolution
+        return change_config_key, check_info, resolution
```

Comment:
```suggestion
    def choose_resolution():
        """
        choose resolution randomly
        """
        n = random.randint(1, 30)
        change_resolution_key = ["kp_enter"] + ["down"] * n + ["kp_enter"]
        resolution = ChangePrefered[n]
        check_info = "GraphicsConsole video resolution " + resolution
        return change_resolution_key, check_info, resolution
```

Reply:
`n = random.randint(1, 30)`should be `n = random.randint(1, 29)`,because ChangePrefered[0] is default set,so resolution could change from ChangePrefered[1] to ChangePrefered[29].Thanks!

Reply:
Yes, you're right. Then I think it should be random.randint(0,29), the default resolution also should be included in the checking list.